src/net/net.c

```diff
@@ -74,6 +74,44 @@ void lwip_status_callback(struct netif *netif)
     rprintf("assigned: %d.%d.%d.%d\n", n[0], n[1], n[2], n[3]);
 }
 
+/* unsigned only ... don't imagine we'd have negative interface numbers! */
+int lwip_atoi(const char *p)
+{
+    u64 i;
+    return u64_from_value(alloca_wrap_cstring(p), &i) ? i : -1;
+}
+
+/* mildly unfortunate to add another level of indirection to resolve types
+   that are equivalent ... maybe a runtime types.h would be in order */
+void lwip_memcpy(void *a, const void *b, unsigned long len)
+{
+    runtime_memcpy(a, b, len);
+}
+
+int lwip_strlen(char *a)
+{
+    return runtime_strlen(a);
+}
+
+void lwip_memset(void *x, unsigned char v, unsigned long len)
+{
+    runtime_memset(x, v, len);
+}
+
+int lwip_memcmp(const void *x, const void *y, unsigned long len)
+{
+    return runtime_memcmp(x, y, len);
+}
+
+int lwip_strncmp(const char *x, const char *y, unsigned long len)
```

Comment:
we also have runtime_strcmp no?

Reply:
Yes, but not strncmp. It should be replaced with runtime_strncmp only for obvious reasons (prob in another PR as it will require some unrelated changes).

The strncmp for lwIP appears to only be used for apps, MS-CHAP and other stuff that's likely not being used.